### PR TITLE
dev-libs/miniaudio: add 0.11.21

### PR DESCRIPTION
**Pull request by stkw0**
Previously the website and other miscellaneous files were also installed in /usr/include. This version corrects that behaviour

### CODE REVIEW

File: dev-libs/miniaudio/miniaudio-0.11.21.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Single file library for audio playback and capture"
+HOMEPAGE="https://miniaud.io"
+SRC_URI="https://github.com/mackron/miniaudio/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( public-domain MIT-0 )"
```
Comment (antecrescent) [inline dev-libs/miniaudio/miniaudio-0.11.21.ebuild:10]:
```suggestion
LICENSE="|| ( Unlicense MIT-0 )"
```
This also affects the other versions in ::gentoo. Maybe you could fix that as well :+1:

**Review by antecrescent — CHANGES_REQUESTED** [review]

**Comment by misanthropos** [comment]
Looks good to me.

**Comment by stkw0** [comment]
> Please make sure the commit message contains the PR description so that information isn't lost

Done. I also rephrase it to make it more clear with the next : `Correct installation of website and miscellaneous files. Now are installed to /usr/share instead of /usr/include.`
